src/com/mendix/widget/badge/Badge.xml

```diff
@@ -0,0 +1,80 @@
+        </property>
+        <property key="badgeType" type="enumeration" required="true" defaultValue="badge">
+            <caption>Type</caption>
+            <category>Display</category>
+            <description>Render either the display of a label or of a badge</description>
+            <enumerationValues>
+                <enumerationValue key="badge">badge</enumerationValue>
+                <enumerationValue key="btn">badge button</enumerationValue>
+                <enumerationValue key="label">label</enumerationValue>
+            </enumerationValues>
+        </property>
+        <property key="badgeClass" type="enumeration" required="true" defaultValue="default">
+            <caption>Style</caption>
+            <category>Display</category>
+            <description>The styling and color of the badge/label using Bootstrap classes.</description>
+            <enumerationValues>
+                <enumerationValue key="default">Default</enumerationValue>
+                <enumerationValue key="primary">Primary</enumerationValue>
+                <enumerationValue key="success">Success</enumerationValue>
+                <enumerationValue key="info">Info</enumerationValue>
+                <enumerationValue key="warning">Warning</enumerationValue>
+                <enumerationValue key="danger">Danger</enumerationValue>
+            </enumerationValues>
+        </property>
+        <property key="attrStyle" type="attribute" required="false">
+            <caption>Bootstrap Style</caption>
+            <category>Data source</category>
+            <description>Value of attribute should be: info, warning, danger, success. Alternatively use the 'Class bar' property set styling</description>
+            <attributeTypes>
+                <attributeType name="Enum" />
+                <attributeType name="String" />
+            </attributeTypes>
+        </property>
+        <property key="attrLabel" type="attribute" required="false">
+            <caption>Label attribute</caption>
+            <category>Data source</category>
+            <description>The attribute that contains the label for the widget</description>
+            <attributeTypes>
+                <attributeType name="Enum" />
+                <attributeType name="String" />
+            </attributeTypes>
+        </property>
+        <property key="attrValue" type="attribute">
+            <caption>Data attribute</caption>
+            <category>Data source</category>
+            <description>The attribute that contains the label/badge value</description>
+            <attributeTypes>
+                <attributeType name="String" />
+                <attributeType name="Integer" />
+                <attributeType name="Decimal" />
+                <attributeType name="Long" />
+            </attributeTypes>
+        </property>
+        <property key="onclickMicroflow" type="microflow" required="false">
+            <caption>Click MicroFlow</caption>
```

Comment:
The second word should not be starting with a capital, and microflow is also not camel cased.